javascript/modules/client/src/internal/interfaces/client.ts

```diff
@@ -73,3 +75,24 @@ export type DaoDepositStepValue =
   | { key: DaoDepositSteps.INCREASED_ALLOWANCE; allowance: bigint }
   | { key: DaoDepositSteps.DEPOSITING; txHash: string }
   | { key: DaoDepositSteps.DONE; amount: bigint };
+
+// DAO DETAILS
+export type DaoResourceLink = { label: string; url: string };
+
+export type DaoToken = {
+  address: string;
+  decimals: number;
+  name: string;
+  symbol: string;
+};
```

Comment:
Given the above points, I'm not sure if this should belong to the ERC20 governance plugin